config/locales/rails_admin.en.yml

```diff
@@ -127,6 +127,7 @@ en:
       save: "Save"
       save_and_add_another: "Save and add another"
       save_and_edit: "Save and edit"
+      delete_file: "Delete '%{object_label}' #%{object_number}"
```

Comment:
Those interpolation names are misleading. `object` in RailsAdmin stands for a model instance to be edited or deleted in member actions, but what's actually here is a field's label, not an object's label. Also `object_number` is not the number of an object, it's the number of an attached file.
Hence I propose `"Delete '%{field_label}' #%{number}"`.

Reply:
Ah, I missed tweaking this after the first review - nice catch 🙇🏼